src/sonic-ctrmgrd/ctrmgr/kube_commands.py

```diff
@@ -487,18 +487,18 @@ def _do_clean(feat, current_version, last_version):
     err = ""
     out = ""
     ret = 0
-    DOCKER_ID = "docker_id"
+    IMAGE_ID = "image_id"
     REPO = "repo"
     _, image_info, err = _run_command("docker images |grep {} |grep -v latest |awk '{{print $1,$2,$3}}'".format(feat))
     if image_info:
         version_dict = {}
         version_dict_default = {}
```

Comment:
one suggestion: use a meaningful name for var, it will be more readable.

Reply:
Fixed